docs/man_pages/project/creation/sample-native.md

```diff
@@ -0,0 +1,31 @@
+sample native
+==========
+
+Usage | Syntax
+------|-------
+General | `$ appbuilder sample native`    
+
+Lists all available NativeScript sample apps with name, description, GitHub repository, and clone command. To clone a selected sample app,
+run its clone command as its listed.
+<% if(isHtml) { %> 
+
+### Related Commands
+
+Command | Description
+----------|----------
+[cloud export](cloud-export.html) | Exports one of your projects from the cloud and initializes it for development
+in the current directory.
+[cloud](cloud.html) | Lists all projects associated with your Telerik Platform account.
+[create hybrid](create-hybrid.html) | Creates a new project from an Apache Cordova-based template.
+[create native](create-native.html) | Creates a new project from a NativeScript-based template.
+[create website](create-website.html) | Creates a new project from a Mobile Website-based template.
+[create](create.html) | Creates a project for hybrid or native development.
+[init hybrid](init-hybrid.html) | Initializes an existing Apache Cordova project for development in the current directory.
+[init native](init-native.html) | Initializes an existing NativeScript project for development in the current directory.
+[init website](init-website.html) | Initializes an existing Mobile Website project for development in the current directory.
+[init](init.html) | Initializes an existing project for development.
+[sample](sample.html) | Lists all available sample apps.
+[sample hybrid] (sample-hybrid.html) | Lists all available Cordova sample apps.
```

Comment:
Again the extra spaces. And please, change all "available Cordova sample apps" to "available Apache Cordova sample apps"